### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by marwin89**
add link to milestones and game design document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,31 +25,32 @@ Supported Operating Systems / Requirements:
...
 
 
-## 2. Download                
+## 2. Download
 You can download precompiled Windows binaries at here:
 https://vocaluxe.org/#download
 
 ## 3. Contributing
 You want to fix a bug or add a cool new feature to Vocaluxe? Awesome!
+Check out [milestones](https://github.com/Vocaluxe/Vocaluxe/milestones) or our [Game Design Document](https://docs.google.com/document/d/1mNn92ezVH63pVVJoIqN1hjmWTX0L_jLb6bPYtGTl8EQ/edit?tab=t.0#heading=h.5p8a2hpxbech)!
```
Comment (flokuep) [inline README.md]: Should be changed as soon as #724 is merged.

File: README.md [hunk 2]
```diff
@@ -25,31 +25,32 @@ Supported Operating Systems / Requirements:
...
-The development of Vocaluxe uses this git branching model:   
+The development of Vocaluxe uses this git branching model:
 http://nvie.com/posts/a-successful-git-branching-model/
 
 ### Questions
-You have a problem or a question implementing your change?   
+You have a problem or a question implementing your change?
 Just [open an issue](https://github.com/Vocaluxe/Vocaluxe/issues/new) or mention `@Vocaluxe` in a comment on GitHub.
 
 ## 4. Help & Support
```
Comment (flokuep) [inline README.md:51]: Maybe add discord here?

**Review by flokuep — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marwin89) [reply README.md]: link is updated ✅

**Comment by marwin89** [comment]
@flokuep : link is updated please merge

**Review by flokuep — APPROVED** [review]

**Pull request merged**